src/workloads/execution/SinusoidalReadWrites.yml

```diff
@@ -0,0 +1,123 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/server-execution"
+Description: |
+  This workload attempts to create a sinusoidal distribution of reads/writes to test
+  the behavior of execution control when the load on the system changes over time.
+
+  We do this with two actors, one for writes and one for reads and spawn 5000 client
+  threads. Though the actors run concurrently, we use the sleep functionality
+  to effectively stop work on one actor while the other reaches its peak. We then
+  alternate for a couple repetitions.
+
+  In this workload, we try to ensure that reads and writes do not overlap much.
+Keywords:
+- sinusoidal
+- execution control
+- insertMany
+- find
+- CrudActor
+
+dbname: &dbname mixed_sinusoidal
+DocumentCount: &NumDocs 5000
+CollectionCount: &NumColls 1
+
+NumPhases: &NumPhases 2
+Document: &doc
+  a: {^RandomInt: {min: 0, max: *NumDocs}}
+  b: {^RandomString: {length: 16}}
+Clients:
+  Default:
+    QueryOptions:
+      maxPoolSize: 5100
+
+ActorTemplates:
+- TemplateName: WritesActorWithSleepTemplate
+  Config:
+    Name: {^Parameter: {Name: "Name", Default: WritesActorWithSleep}}
+    Type: CrudActor
+    Database: *dbname
+    Threads: {^Parameter: {Name: "Threads", Default: 1000}}
+    Phases:
+      OnlyActiveInPhases:
+        Active: {^Parameter: {Name: "ActivePhases", Default: [1]}}
+        NopInPhasesUpTo: *NumPhases
+        PhaseConfig:
+          Duration: 3 minutes
+          SleepAfter: 90 seconds
```

Comment:
Once we get confirmation via the intra-run data that the sleep does what we want, I think we can parameterize this option. This way, we can reuse this actor for subsequent workloads.

We can do this for SleepAfter, SleepBefore, and Repeat.